docs/source/projections/eqc.rst

```diff
@@ -8,3 +8,23 @@ Equidistant Cylindrical (Plate Caree)
    :scale: 50%
    :alt:   Equidistant Cylindrical (Plate Caree)  
 
+The simplist of all projections. Standard parallels (0° when omitted) may be specified that determine latitude of true scale (k=h=1).
+
+.. math::
+
+   x = \lambda cos \phi_{ts}
```

Comment:
Not specific to @julien2512's contribution (it is greatly appreciated!), but this highlights the tricky part about this endeavor -- it is really easy to miss math symbols when porting things over.